low-level/sensors/impl/generic/node_hw_alerts_info.py

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) 2001-2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+from collections import namedtuple
+from framework.base.sspl_constants import DEFAULT_RECOMMENDATION
+
+Alert = namedtuple('AlertInfo',
+                   ['type', 'severity', 'description',
+                    'impact', 'recommendation'])
+
+alert_for_event = {
+    "Fan": {
```

Comment:
Is it possible to make it generic for all threshold-based senors @gargpalak048 ❓    
The events seem to be the same for all. 
Else looks great to me. 👍

Reply:
I kept a tuple per event to make it more loosely bound.
If we need to modify any field for particular event, we could do that.

Reply:
But yes, we can do something like,

> **dict.fromkeys(["Sensor threshold event", "Sensor threshold event ()"], {

            "Asserted": Alert(
                "fault", "critical",
                "{event-name} threshold breached for '{}'. [{}]",
                "Fan speed is outside acceptable limits",
                DEFAULT_RECOMMENDATION
            ),
   
then it will be tightly bound to all the events.
What could be the bettter approach?

Reply:
No, no. not about events, I was trying to cover all the threshold-based sensors with the same key, so if we can have a single key for all threshold-based sensors (eg. "Threshold-based-sensor") for Fan, temp, voltage, current, etc, instead of having a separate key for each of them.
But not needed now, I think. If such optimization is required we can handle it later.